app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,48 @@
+<h2><%= @recipe.name %></h2>
+<div>
+  <h4>
+    Preparation time: 
+    <span>
+      <%= @recipe.preparetion_time %>
+    </span>
+  </h4>
+
+  <%= form_with do |form| %>
+    <%= form.label :public, "Public:" %>
+    <%= form.check_box :public %>
+  <% end %>
```

Comment:
- [ ] [REQURED] the public value is not displayed when true, you could use  form_for @recipe
- [ ] [OPTIONAL] make the checkbox disabled to prevent confusion on the users part